cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/BasicMapReduceContext.java

```diff
@@ -356,7 +355,7 @@ public File getPluginArchive() {
    * as a part of it, otherwise {@code null} is returned.
    */
   @Nullable
-  public WorkflowProgramInfo getWorkflowProgramInfo() {
+  public WorkflowProgramInfo getWorkflowInfo() {
```

Comment:
Oh right. Thought IDE had done that for me... I just merged it, not seeing your comment. I guess I can amend the commit if that's your only comment?